lib/Zef/Repository/Ecosystems.pm6

```diff
@@ -37,9 +37,10 @@ class Zef::Repository::Ecosystems does Repository {
             UNDO note "!!!> Failed to update $!name mirror: $uri";
             KEEP note "===> Updated $!name mirror: $uri";
             KEEP self!gather-dists;
+            CATCH { .note; }
 
             my $save-as  = $!cache.IO.child($uri.IO.basename);
-            my $saved-as = try $!fetcher.fetch(Candidate.new(:$uri), $save-as, :timeout(180));
+            my $saved-as = $!fetcher.fetch(Candidate.new(:$uri), $save-as, :timeout(180));
```

Comment:
You need to add the `try` back here and put the `CATCH` inside the `try` block. Otherwise you are effectively wrapping this entire block inside a`try` and changing the behavior of any existing `try`. Additionally you should use `CATCH { default { .note } }`